src/index.html

```diff
@@ -2,16 +2,218 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application" method = "post">
+
+      <fieldset class = "container personal-information">
+        <legend>Personal information</legend> 
+        <div class="form-field">
+          <label>
+            Surname:
+          </label>
+
+          <input 
+           type="text" 
+           name="Surname"
+           required
+           autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label>
+            Name:
+          </label>
+
+          <input 
+          type="text" 
+          name="Name"
+          required
+          autocomplete="off"
+          >
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) the 3rd point
```suggestion
          <input 
	          type="text" 
	          name="Name"
	          required
	          autocomplete="off"
          >
```